### PR TITLE
Modify guides to include information that describes how to obtain Ref…

### PR DESCRIPTION
**Pull request by speedyman08**
…resher from the github

### CODE REVIEW

File: Docs/topics/patching/rpcs3.md [hunk 1]
```diff
@@ -2,9 +2,19 @@
 
 Refresher has native support for decrypting and patching RPCS3 games. This guide aims to show you how to operate Refresher's RPCS3 patcher. At the end of this, you will be able to select your server of choice inside the game patches menu of RPCS3.
 
-It also assumes that you've already downloaded Refresher. If you haven't, head on over to the [GitHub page](https://github.com/LittleBigRefresh/Refresher) to grab yourself a copy.
+To download Refresher, you must first visit the [Github Page](https://github.com/LittleBigRefresh/Refresher). Once you have opened it in your browser,
+navigate to the "Releases" section on the website.
 
-![A Screenshot of Refresher's RPCS3 patching screen](refresher-rpcs3.png)
```
Comment (jvyden) [inline Docs/topics/patching/rpcs3.md:7]: Why is this line removed? Mistake?

File: Docs/topics/patching/ps3.md [hunk 2]
```diff
@@ -2,15 +2,25 @@
...
+
+Once you have downloaded the built binaries for your operating system,
+simply unzip the archive, and you should have an executable.
 
 ## Step 1: Jailbreak your PS3 and install webMAN
 
 We can't help you with this, but there are various guides out there you can go out and find. We personally recommend MrMario2011's guides.
 
-In general, though, try to use the latest firmware, use HEN for super slims, CFW for other models, etc., etc.
+In general, though, try to use the late[s]()t firmware, use HEN for super slims, CFW for other models, etc., etc.
```
Comment (jvyden) [inline Docs/topics/patching/ps3.md]: ?

File: Docs/topics/patching/ps3.md [hunk 3]
```diff
@@ -2,15 +2,25 @@
 
 Refresher has native support for decrypting, encrypting and patching PS3 games. This guide aims to show you how to operate Refresher's PS3 patcher.
 
-It also assumes that you've already downloaded Refresher. If you haven't, head on over to the [GitHub page](https://github.com/LittleBigRefresh/Refresher) to grab yourself a copy.
+To download Refresher, you must first visit the [Github Page](https://github.com/LittleBigRefresh/Refresher). Once you have opened it in your browser, 
+navigate to the "Releases" section on the website.
 
-![A Screenshot of Refresher's PS3 patching screen](refresher-ps3.png)
```
Comment (jvyden) [inline Docs/topics/patching/ps3.md:7]: Same here

**Review by jvyden — CHANGES_REQUESTED** [review]

File: Docs/topics/patching/ps3.md (hunk 2 above)
Comment (speedyman08) [reply Docs/topics/patching/ps3.md]: I have no idea how that happened. Don't ever remember touching that line, i'll fix it now

**Review by jvyden — APPROVED** [review]
